app/views/tweets/new.erb

```diff
@@ -0,0 +1,13 @@
+<h1>New Tweet:</h1>
+
+<form action="/tweets" method="POST">
+  <input type="hidden" name="user_id" id="hidden" value="<%=current_user.id%>" />
```

Comment:
I would recommend against doing this. Even though the field is hidden, it can be manipulated through the browser, allowing a malicious user to create a tweet under a different user's account. Instead, associate the new tweet with the `current_user` in the controller, relying on the session instead of the form field.